src/docs/guides/decoupled-cms/README.md

```diff
@@ -324,7 +324,7 @@ We can now update the Razor Page to use the alias instead of the content item id
 }
 ```
 
-!!! waring "Release packages"
+!!! wanring "Release packages"
```

Comment:
I'll call Frodo to take care of the wan ring :).

Reply:
My precious.